### DISCUSSION TITLE
Acquire Devices Rendered Template

### DISCUSSION THREAD

**Discussion by danner26** [netbox-community/pynetbox#553] [Q&A] [answered]
Hello,

Is pynetbox able to retrieve a devices rendered config tempalte yet?

**Comment by v0tti** [comment]
I would be interested in this as well. At the moment it does not seem possible.

**Reply by jhofmueller** [reply]
According to this post in netbox discussions it should work but the documentation doesn't get rendered:

https://github.com/netbox-community/netbox/discussions/13800

**Reply by jhofmueller** [reply, marked as answer]
Here is how it works:

```python
>>> import pynetbox
>>> nb = pynetbox.api(
...     'http://localhost:8000',
...     token='d6f4e314a5b5fefd164995169f28ae32d987704f'
... )
>>> nb.dcim.devices.get(<filter>).render_config.create()
```
This returns a dictionary containing among other things the key `content` which is the rendered config.
